src/guest/start.c

```diff
@@ -44,7 +44,6 @@ static const char *fixed_cmd =
 	" -device qemu-xhci,id=xhci,p2=8,p3=8"
 	" -device usb-mouse"
 	" -device usb-kbd"
-	" -device e1000,netdev=net0"
```

Comment:
why remove e1000 device?

Reply:
While launching CIV ,If we emulate net configuration by default along  with Ethernet passthrough ,android fastboot will generate IPV6 address based on emulated net configuration which results in fastboot connection failure.
And if we remove net configuration by default and we are not passing through Ethernet it will result in Internet connection failure.

We need to add below lines in the cmd section of vm manger ini for the Internet when we don't passthrough internet

cmd= -device e1000,netdev=net0

Reply:
In start CIV , dynamically adds / remove e1000 device based on ethernet passthrough . But in VM manager its always enabled which is why there are failure in generation of MAC address in fastboot.